lib/quiz/sequence.dart

```diff
@@ -31,93 +33,88 @@ class SequenceQuizState extends State<SequenceQuiz> {
   }
 
   void _initboard() {
+
+    // Array for storing choices from input
     for (var i = 0; i < widget.input['order'].length; i++) {
       choice.add(widget.input['order'][i]);
     }
     // choice = choice.map((a) => widget.input['order'][a]).toList(growable: false);
     ans = widget.input['image'];
     print("Choices at initializtion -$choice");
+    
+    // Array to track the choices clicked
+    clicked = choice.map((e) => "false").toList(growable: false);
```

Comment:
Do you want the String 'false' or bool 'false'? It is dangerous to rely on strings when all you need is a boolean value

Reply:
I'm not just relying on a Boolean value. I'm later setting the value of clicked array to "completed" so as to inform that all the buttons have been clicked and its time to display options in green/red